### DISCUSSION TITLE
How to call virtual agent to get a "self-serve refunds for Copilot"?

### DISCUSSION THREAD

**Discussion by TuanDang293** [community/community#66077] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi, I have recently cancelled my Copilot subscription due to company policy. However, I still got charged $10 yesterday while VS Code extension gave me notification that my copilot expired. Therefore, I want to get a refund. I saw in the discussion that I could do that through the Virtual Agent, but I could not call it in the Help with your GitHub account page. I tried "refund copilot" and similar prompt but all I got was manage your receipt with Virtual Agent.

**Comment by Akash1134** [comment, marked as answer]
Hi @TuanDang293 The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by denverharry** [comment]
I also want to know because i want to call [Concrete Foundations in Kingsport TN](https://www.kingsportconcreteservices.com/concrete-foundations-kingsport-392439.html) website agent.

**Comment by VladyslavHerasymiuk** [comment]
Hi, I have the same issue. Do you get your refund? Could you share your experience?

**Reply by TuanDang293** [reply]
Send in a support ticket and wait. I got it after a couple days.
